hoot-core/src/main/cpp/hoot/core/conflate/ConflateCmd.cpp

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
No, those files shouldn't be there at all.  The correct files are core/cmd/ConflateCmd.cpp/h.  Those two must have been copied in there by accident.